src/SDK/vector.h

```diff
@@ -1351,9 +1351,9 @@ inline void QAngle::Init(float ix, float iy, float iz)
 
 inline void QAngle::Random(float minVal, float maxVal)
 {
-	x = minVal + ((float)rand() / RAND_MAX) * (maxVal - minVal);
-	y = minVal + ((float)rand() / RAND_MAX) * (maxVal - minVal);
-	z = minVal + ((float)rand() / RAND_MAX) * (maxVal - minVal);
+	x = minVal + ((float)rand() / (float)RAND_MAX) * (maxVal - minVal);
```

Comment:
if you wanna be proper about this, you should use VALVE_RAND_MAX instead, and apply it to the other ones too

Reply:
@LWSS This I should fix...